shuup/reports/writer.py

```diff
@@ -122,6 +124,51 @@ def get_filename(self, report):
         return "%s%s" % ((report.filename_template % fmt_data).replace(":", "_"), self.extension)
 
 
+class CSVReportWriter(ReportWriter):
+    content_type = "text/csv"
+    extension = ".csv"
+    writer_type = "csv"
+
+    def __init__(self):
+        super(CSVReportWriter, self).__init__()
+        self.data = []
+
+    def _w(self, content):
+        if content is not None:
+            if isinstance(content, TaxlessPrice) or isinstance(content, TaxfulPrice):
+                content = floatformat(content.amount.value, 2)
+
+            if isinstance(content, Decimal):
+                content = floatformat(content, 2)
```

Comment:
idk, looks wrong to set a fixed number of decimal places here without knowing what it is about. what if:
```py
>>> from decimal import Decimal
>>> value = 32.24738
>>> Decimal(str(value)).normalize().as_tuple().exponent
-5
```
then you can get this exponent and use in the floatformat, use some sort of max() so you prevent using a huge precision, set max as 10?